Failing with TS1371 using generated routes for Koa

While generating routes for Koa I'm getting the following error while building with `tsc` or running with `ts-node`.

```
error TS1371: This import is never used as a value and must use 'import type' because 'importsNotUsedAsValues' is set to 'error'.

import * as KoaRouter from '@koa/router';
```

## Sorting

- **I'm submitting a ...**

  - [x] bug report
  - [ ] feature request
  - [ ] support request

- I confirm that I
  - [x] used the [search](https://github.com/lukeautry/tsoa/search?type=Issues) to make sure that a similar issue hasn't already been submit

## Expected Behavior

I would expect a type import as generated in the line above so it would be:

```ts
import type * as KoaRouter from '@koa/router';
```

## Current Behavior

Currently it's generating:

```ts
import * as KoaRouter from '@koa/router';

```

## Context (Environment)

Version of the library: 4.1.3
Version of NodeJS: v18.12.1

- Confirm you were using yarn not npm: [x]

## Breaking change?

Should not be a breaking change

--- a/packages/cli/src/routeGeneration/templates/koa.hbs
+++ b/packages/cli/src/routeGeneration/templates/koa.hbs
@@ -24,7 +24,7 @@ import { iocContainer } from '{{iocModule}}';
 import { IocContainer, IocContainerFactory } from '@tsoa/runtime';
 {{/if}}
 import type { Middleware } from 'koa';
-import * as KoaRouter from '@koa/router';
+import type * as KoaRouter from '@koa/router';
 {{#if useFileUploads}}
 const multer = require('@koa/multer');
 const upload = multer({{{json multerOpts}}});

--- a/packages/cli/src/routeGeneration/templates/koa.hbs
+++ b/packages/cli/src/routeGeneration/templates/koa.hbs
@@ -21,7 +21,7 @@ const promiseAny = require('promise.any');
 {{/if}}
 {{#if iocModule}}
 import { iocContainer } from '{{iocModule}}';
-import { IocContainer, IocContainerFactory } from '@tsoa/runtime';
+import type { IocContainer, IocContainerFactory } from '@tsoa/runtime';
 {{/if}}
 import type { Middleware } from 'koa';
 import type * as KoaRouter from '@koa/router';

